jellyfin_kodi/database/jellyfin_db.py

```diff
@@ -71,6 +71,15 @@ def get_item_by_kodi_id(self, *args):
         except TypeError:
             return
 
+    def get_episode_kodi_parent_path_id(self, *args):
+
+        try:
+            self.cursor.execute(QU.get_episode_kodi_parent_path_id, args)
+
+            return self.cursor.fetchone()[0]
+        except TypeError:
+            return
```

Comment:
I agree with the github scanner about not mixing implicit and explicit returns.  I believe in this case it's better to `return 0` (or `None` explicitly, but imo keeping the same type is generally better practice when feasible) and check the return value before attempting to use it on the calling function.

Reply:
I was thinking the same, but this is the code style currently used in these files, so I don't consider it a blocker, cleaner to just fix them all in one go later some time.